cmd/download.go

```diff
@@ -26,6 +26,38 @@ import (
 var DefaultParallelization = int(float32(runtime.NumCPU()) * 0.8) // Default to 80% of available cores
 const MaxRequeues = 3
 
+// splitVersion will split a version string into an X.Y string and a Z string
+func splitVersion(version string) (xy, z string) {
+	if version != "" {
+		r := strings.Split(version, ".")
+		xy = r[0] + "." + r[1]
+		z = r[2]
+	}
+	return
+}
+
+// deprecatedHubVersionToAcmMce translates the hubVersion to acmVersion and mceVersions to provide
+// minimal backwards-compatible support for the deprecated --hub-version option, which uses an invalid
+// assumption that the Z value of the ACM and MCE Z-stream versions are the same. Because this assumption
+// is invalid, the --hub-version is replaced with separate --acm-version and --mce-version options.
+// This function uses the previous (invalid) translation in order to support users that have not updated
+// to using the new options.
+func deprecatedHubVersionToAcmMce(hubVersion string) (acmVersion, mceVersion string) {
+	acmVersion = hubVersion
+
+	xy, z := splitVersion(hubVersion)
+
+	if xy == "2.6" {
+		mceVersion = "2.1." + z
+	} else if xy == "2.5" {
```

Comment:
should we include a new if-then-else for ACM 2.7.z = MCE 2.2.z. Just in case this feature is released once ACM 2.7 is out.

Reply:
Users will be able to use the explicit --mce-version at that point. The deprecated --hub-version should only need to cover the current configuration status